### DISCUSSION TITLE
Integrazione per Living Now Bticino

### DISCUSSION THREAD

**Discussion by jovannig** [indomus/forum#1971] [HUB: Home Assistant]
Ho un po' di confusione in testa.
Ho installato la serie Living Now in casa, interruttori e prese connessi, misuratori di corrente e varie.
Ho trovato come integrazione su HA sia Legrand che Bticino e Netatmo.
Quale devo utilizzare per sfruttare la line Living Now?
Grazie a tutti!

**Comment by essegi65** [comment]
Non conosco quelle integrazioni, io uso un sistema "misto" per living now. Ovvero interruttori luci con il gateway bticino integrati in HA tramite Controller homekit, mentre le poche prese comandate tramite zigbee (così riesco a vedere anche i consumi, cosa che homekit al momento non permette)

**Comment by jovannig** [comment]
Io sto provando oggi l'integrazione bticino. Funziona anche per i consumi, ma non sono in tempo reale. In piu' non riesco a connettere il contatore generale dei consumi. Te come hai fatto? Hai resettato le prese e le hai ricollegate con zigbee a HA? E con il contatore generale come hai fatto a collegarlo a HA?

**Reply by essegi65** [reply]
Esatto resetta le prese (togliendole dal gateway bticino) e fai il paring con zigbee. Per i consumi totali io non ho nessun elemento bticino ma uso uno shelly em, attaccato proprio sotto il generale e integrato in HA. La lettura dei consumi è immediata in con entrambe le tipologie

**Reply by jovannig** [reply]
Ciao, dopo un po' di peripezie sono finalmente riuscito ad installaze ZHA e a riconoscere i misuratori DIN.
Prima li ho resettati tenendo premuto il tasto reset finche non lampeggiavano di rosso, dopodiche' li ho fatti riconoscere a ZHA.
Lui li vede senza problemi, ma tutti i valori dei consumi rimangono a zero, non so come fare e non capisco dove sbaglio. Forse devo togliere la corrente alla casa e riprovare?

**Reply by jovannig** [reply]
Sono riuscito a risolvere, bastava ripremere sul tasto reset appena compare la spia verde lampeggiante

**Comment by jovannig** [comment]
Ho provato a resettare il modulo gestione carichi FC80GCS ma non viene riconoscito da Zigbee2Mqtt, mi da' unsupported come status. Non so come fare a risolvere

**Comment by jovannig** [comment]
Ho provato a resettare anche una presa connessa K4531C ma zigbee2mqtt mi da' unsupported e su HA non la vedo ne' nei devices ne' nelle entities

**Reply by essegi65** [reply]
Cosa usi come chiavetta zigbee? [qui](https://zigbee.blakadder.com/BTicino_4531C.html) puoi vedere che il prodotto è supportato io lo uso con ZHA. Prova a cambiare canale di zigbee quando ho fatto il paring la prima volta non ne voleva sapere ho messo il canale 11 e ci sono riuscito. Per gli altri modelli mi dispiace ma non li uso e non so aiutarti.

**Reply by jovannig** [reply]
Io uso HA come container su docker a prima avevo un container con zigbee2mqtt.
Ho disinstallato zigbee2mqtt ma adesso non so come fare per creare un container con ZHA.
Se provo ad installare ZHA da HA, mi blocco quando mi chiede la porta della chiavetta Conbee II, non so come andare avanti.
Se riuscissi ad installare ZHA poi ho visto che i due DIN che ho sono compatibili.

**Reply by essegi65** [reply]
Se non vado errato la porta della chiavetta la dovevi inserire anche con Z2M, ma qui mi fermo io non sono così esperto d'installazione su doker. Aspetta che qualcuno di aiuti per l'installazione di ZHA. Poi se dopo ti occorre altro aiuto per la configurazione di living now posso aiutarti.

**Reply by jovannig** [reply]
Grazie. Spero che qualcuno mi dia una mano per ZHA, perche' avendo i container non so come mappare la porta /dev/ttyACM0 in modo da farla usare da HA, tramite docker compose

**Comment by jovannig** [comment]
L'unica periferica che zigbee2mqtt e' stato in grado di riconoscere e' il Living Now Misuratore DIN Connesso F20T60A.
Ne ho installati 2 e per entrambi, su HA le 3 entities che sono state generate: sensor, power_alarm e power_alarm_active hanno come stato "unknown". Qualcuno sa darmi una mano?
